### DISCUSSION TITLE
Feature Request: Allow entityID as rule/rollout segment

### DISCUSSION THREAD

**Discussion by graytonio** [flipt-io/flipt#2069] [Ideas]
When creating an evaluation rule or rollout for a flag we are able to select a segment as the "target" of the rule and in most cases this is perfect. However in some circumstances I want to be able to have a flag set for a specific entityID especially if that ID is something meaningful like an email or username like the docs mention. 

I think this would be even less processing than a segment rule as there is not determining if an entity belongs to a segment or not since it is only determined by the entityID.

**Comment by markphelps** [comment]
Hi @graytonio! Love the idea and can definitely see the use case of having 'one off' entityID matches without requiring creating segments.

It would be a pretty large shift in our data model however so would just need to do some thinking on how to best design it architecturally (db mostly) and how the UI might change. Will discuss with the rest of the team on Monday, but also feel free to add any of your thoughts on either here.
